tests/scripts/all.sh

```diff
@@ -788,63 +787,78 @@ component_build_default_make_gcc_and_cxx () {
 }
 
 component_test_submodule_cmake () {
-    # USE_CRYPTO_SUBMODULE: check that the build works with CMake
-    msg "build: cmake, full config + USE_CRYPTO_SUBMODULE, gcc+debug"
+    # USE_CRYPTO_SUBMODULE: check that it's enabled by default with cmake
+    msg "build: cmake, full config (with USE_CRYPTO_SUBMODULE), gcc+debug"
     scripts/config.pl full # enables md4 and submodule doesn't enable md4
-    scripts/config.pl unset MBEDTLS_MEMORY_BACKTRACE # too slow for tests
-    CC=gcc cmake -D USE_CRYPTO_SUBMODULE=1 -D CMAKE_BUILD_TYPE=Debug .
+    CC=gcc cmake -D CMAKE_BUILD_TYPE=Debug .
     make
     msg "test: top-level libmbedcrypto wasn't built (USE_CRYPTO_SUBMODULE, cmake)"
     if_build_succeeded not test -f library/libmbedcrypto.a
     msg "test: libmbedcrypto symbols are from crypto files (USE_CRYPTO_SUBMODULE, cmake)"
     if_build_succeeded objdump -g crypto/library/libmbedcrypto.a | grep -E 'crypto/library$' > /dev/null
     msg "test: libmbedcrypto uses top-level config (USE_CRYPTO_SUBMODULE, cmake)"
     if_build_succeeded objdump -g crypto/library/libmbedcrypto.a | grep 'md4.c' > /dev/null
-    msg "test: main suites (USE_CRYPTO_SUBMODULE, cmake)"
-    make test
-    msg "test: ssl-opt.sh (USE_CRYPTO_SUBMODULE, cmake)"
-    if_build_succeeded tests/ssl-opt.sh
+    # no need for runtime tests - this is the default, tested elsewhere
+
 }
 
 component_test_submodule_make () {
-    # USE_CRYPTO_SUBMODULE: check that the build works with make
-    msg "build: make, full config + USE_CRYPTO_SUBMODULE, gcc+debug"
+    # USE_CRYPTO_SUBMODULE: check that it's enabled by default with make
+    msg "build: make, full config (with USE_CRYPTO_SUBMODULE), gcc+debug"
     scripts/config.pl full # enables md4 and submodule doesn't enable md4
-    scripts/config.pl unset MBEDTLS_MEMORY_BACKTRACE # too slow for tests
```

Comment:
Was it intentional to remove the unsetting of `MBEDTLS_MEMORY_BACKTRACE`?

Reply:
Yes, as it's only needed if we execute the test suites (to avoid that being too slow).